src/styles/_catalog.scss

```diff
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: $transition-duration;
```

Comment:
Previous comment not fixed. Please specify for which property transition should be applied
https://developer.mozilla.org/en-US/docs/Web/CSS/transition